src/test/java/org/cactoos/io/TeeInputFromBytesTest.java

```diff
@@ -51,40 +51,43 @@ public void copiesFromBytesToPath() throws IOException {
         final String message =
             "Hello, товарищ path äÄ üÜ öÖ and ß";
         final File output = this.folder.newFile();
-        MatcherAssert.assertThat(
+        new Assertion<>(
+            "Bytes to path test - Message content is required",
```

Comment:
@marceloamadeu usually in the cactoos project, we always use the form: "Must ..." followed by what the object under test must do. Here it would be: "Must copy bytes to file path". Can you fix it here and in the other that you added?

Reply:
@victornoel Done ... please check.